doc/classes/TextEdit.xml

```diff
@@ -1154,7 +1154,7 @@
 			<return type="void" />
 			<param index="0" name="enabled" type="bool" />
 			<description>
-				If [code]true[/code], sets the user into overtype mode. When the user types in this mode, it will override existing text.
+				If [code]true[/code], sets the user into overtype mode. When the user types in this mode, it will override existing text. The [member ProjectSettings.input/ui_text_toggle_insert_mode] action toggles overtype mode. See [method is_overtype_mode_enabled].
```

Comment:
I think "it" is referring to the TextEdit, not the user. Since the user types, but the TextEdit is what overrides existing text.
That sentence can be rewritten to something like:
"In this mode, typing overrides existing text instead of inserting text."

Reply:
The getter equivalent of this method also refers to "the user" so if you go that route that should be changed too.